ibis/expr/tests/test_table.py

```diff
@@ -1030,23 +1030,56 @@ def test_join_nontrivial_exprs(table):
     assert False
 
 
-def test_union(table):
-    schema1 = [('key', 'string'), ('value', 'double')]
-    schema2 = [('key', 'string'), ('key2', 'string'), ('value', 'double')]
-    t1 = ibis.table(schema1, 'foo')
-    t2 = ibis.table(schema1, 'bar')
-    t3 = ibis.table(schema2, 'baz')
-
-    result = t1.union(t2)
+@pytest.fixture
```

Comment:
i'd rather you just inline these as these fixtures are only used in 2 places and not well named / re-usable

Reply:
@jreback I do think that they would be reused in the future, because you're still missing the SQL exception operation. Is there a name that you think would be better?